examples/sphDiagramGeneration/sph-diagram-generation.cpp

```diff
@@ -60,47 +55,30 @@ int main(int argc, char *argv[]) {
 
   int numParticles;
   int numIterations;
-  int containerTypeInt = 0;
-  enum ContainerType { linkedCells, directSum, verletLists, verletListsCells } containerType = linkedCells;
+  std::set<autopas::ContainerOption> containerOptions;
   int functorTypeInt = 0;
   enum FunctorType { densityFunctor, hydroForceFunctor } functorType = densityFunctor;
 
   double skin = 0.;
   int rebuildFrequency = 10;
   bool useNewton3 = true;
-  if (argc == 9) {
-    numParticles = std::stoi(argv[1]);
-    numIterations = std::stoi(argv[2]);
-    containerTypeInt = std::stoi(argv[3]);
-    functorTypeInt = std::stoi(argv[4]);
-    skin = std::stod(argv[5]);
-    rebuildFrequency = std::stoi(argv[6]);
-    useNewton3 = std::stoi(argv[7]);
+  if (argc >= 9) {
     boxMax[0] = boxMax[1] = boxMax[2] = std::stod(argv[8]);
-  } else if (argc == 8) {
-    numParticles = std::stoi(argv[1]);
-    numIterations = std::stoi(argv[2]);
-    containerTypeInt = std::stoi(argv[3]);
-    functorTypeInt = std::stoi(argv[4]);
-    skin = std::stod(argv[5]);
-    rebuildFrequency = std::stoi(argv[6]);
+  }
+  if (argc >= 8) {
     useNewton3 = std::stoi(argv[7]);
```

Comment:
you are filling a bool with `stoi`. Just saying...

Reply:
the only proper alternative is to use boost::lexical_cast, which I don't want to ;)
or writing your own proper parser, which I also don't want to do.

Reply:
and currently we only allow one traversal